Modules/core/BRTransaction.h

```diff
@@ -109,9 +109,9 @@ typedef struct {
 //    char name[31]; // MAX 31 Bytes
     size_t nameLen;
     uint64_t amount;     // 8 Bytes
-    int64_t unit;        // 1 Byte
-    int64_t reissuable;  // 1 Byte
-    int64_t hasIPFS;     // 1 Byte
+    int unit;        // 1 Byte
+    int reissuable;  // 1 Byte
+    int hasIPFS;     // 1 Byte
```

Comment:
```suggestion
    int8_t hasIPFS;     // 1 Byte
```

Reply:
Reason: int may be different sizes depending on the bitness of the machine being used to compile.  int8_t should be a single 8-bit byte regardless of compiler or bitness of the machine.